test/test_reductions.py

```diff
@@ -3128,6 +3128,22 @@ def test_tensor_compare_ops_empty(self, device):
             # Check if function raises error on specified zero'd dimension as reduction dim.
             self.assertRaisesRegex(IndexError, "Expected reduction dim", lambda: fn(master_input, dim=1))
 
+    # Test to ensure that the reduction operators, amax and amix throws errors for zero-dim
+    # tensors (i.e. empty tensors)
+    def test_amax_amin_no_dim_arg(self, device):
+        shape = (2, 0, 4)
+        master_input = torch.randn(shape, device=device)
+        test_functions = [
+            (torch.amax, {'dtype': torch.int64}),
+            (torch.amin, {'dtype': torch.int64})
+        ]
+
+        err_msg = "Specify the reduction dim with the 'dim' argument."
+
+        for fn, dtype in test_functions:
+            with self.assertRaisesRegex(RuntimeError, err_msg):
+                fn(master_input)
```

Comment:
For coverage, it would be nice to have a case with dim argument.

Reply:
Yep! Added. Thanks